cspell-wordlist.txt

```diff
@@ -4,6 +4,7 @@
 !Typescript
 !nodejs
 BUILDID
+bromann
```

Comment:
You mean to add this?

Reply:
I had my handle in the todo message first but removed it, so we can remove this too.